runners/cromwell_on_google/wdl_runner/wdl_pipeline.yaml

```diff
@@ -1,24 +1,9 @@
-name: WDL Runner
-description: Run a workflow defined by a WDL file
-
-inputParameters:
-- name: WDL
-  description: Workflow definition
-- name: WORKFLOW_INPUTS
-  description: Workflow inputs
-- name: WORKFLOW_OPTIONS
-  description: Workflow options
-
-- name: WORKSPACE
-  description: Cloud Storage path for intermediate files
-- name: OUTPUTS
-  description: Cloud Storage path for output files
-
-docker:
-  imageName: gcr.io/broad-dsde-outreach/wdl_runner:2017_10_02
-
-  cmd: >
-    /wdl_runner/wdl_runner.sh
+actions:
+- name: WDL_Runner
+  commands: [ '/wdl_runner/wdl_runner.sh' ]
+  imageUri: gcr.io/google.com/kbinder-sandbox/wdl_runner 
 
 resources:
-  minimumRamGb: 3.75
+   virtualMachine:
+     machineType: n1-standard-2
```

Comment:
Why the change here?  I would have thought we were getting a n1-standard-1 before?  Sorry if I missed something

Reply:
You're right.  Should be n1-standard-1